### PR TITLE
[doc] (root) contribution and templates

### PR DESCRIPTION
**Pull request by dzdidi**
I've decided to keep it in a single file for now. We can add `CONTRIBUTING.md` later on

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,49 @@
...
+3. Ensure test and linting are exectuted succesfully;
+4. You may merge the Pull Request in once you have the sign-off of other developers, or if you do not have permission to do that, you may request the reviewer to merge it for you;
+
+### Commit message template
+Run
+```bash
+git config commit.template  ./.gitmessage
+```
+
+### Adding new asstes
```
Comment (dylanseago) [inline README.md]: typo on `asstes`

File: README.md [hunk 2]
```diff
@@ -2,6 +2,49 @@
...
+#### Importing packages
+Use lerna to import packages (More info in [lerna docs](https://github.com/lerna/lerna/))
+#### Configuring assets
+Add payment factory to `packages/coin-payments/src/constants.ts`
+
+#### Registring assets
+Add payment config to `packages/coin-payments/src/types.ts`
+
+### Abstractions to implement
+Find types and intefraces at `packages/payments-common/src/`
```
Comment (dylanseago) [inline README.md]: typo on `intefraces`

**Comment by dylanseago** [comment]
What do you think about using something like commitizen to enforce the guidelines? http://commitizen.github.io/cz-cli/

**Comment by dzdidi** [comment]
@dylanseago sorry for typos, spellcheck is already enabled. 

I do prefer conventions over configurations, and do not think that we need to do an enforcement now when conventions are so simple and templated. Two of us can agree, I believe, we already did.

Testing, linting and following contribution guide are common sense practices and if someone is not able to follow them, then this person is free to make their local environment to enforce them to do so. 

We already have husky running lint-staged on precommit. Two extra libraries for one simple practice. In general, I do not think we need to add more tools / dev-dependencies, I do think we need to reduce their amount to necessary minimum.

**Comment by dzdidi** [comment]
@dylanseago can it be merged?

**Pull request merged**